libs/language-server/src/lib/services/jayvee-documentation-provider.ts

```diff
@@ -0,0 +1,15 @@
+import { AstNode, findCommentNode } from 'langium';
+import { JSDocDocumentationProvider } from 'langium/lib/documentation/documentation-provider';
+
+export class JayveeDocumentationProvider extends JSDocDocumentationProvider {
+  override getDocumentation(node: AstNode): string | undefined {
+    const lastNode = findCommentNode(
+      node.$cstNode,
+      this.grammarConfig.multilineCommentRules,
+    );
+    if (lastNode === undefined) {
+      return;
+    }
+    return lastNode.text.replace('/*', '').replace('*/', '');
```

Comment:
Can probably written in a more elegant way that does not rely on magic strings but rather pulls them from the AST or so.